src/mbgl/util/stopwatch.hpp

```diff
@@ -4,9 +4,21 @@
 #include <mbgl/util/chrono.hpp>
 
 #include <string>
+#include <sstream>
 
 namespace mbgl {
 namespace util {
+    
+#ifdef MBGL_TIMING
+#define MBGL_TIMING_START(watch)  std::shared_ptr<util::stopwatch> watch = std::make_unique<util::stopwatch>(Event::Timing);
```

Comment:
Using the `shared_ptr` here confused me at first, but I guess the the reason for this is so that we can copy the watch by-value into a lambda function as in `default_file_source`? Now that I think about it, it makes a lot of sense. Maybe just add a comment to help other people who get confused like me:

`// Declare 'watch' as a shared_ptr so it can be captured by value in a lambda function`